crates/core/component/shielded-pool/src/note_payload.rs

```diff
@@ -14,6 +14,7 @@ pub struct NotePayload {
     pub note_commitment: note::StateCommitment,
     pub ephemeral_key: ka::Public,
     pub encrypted_note: NoteCiphertext,
+    pub sync_exclude: bool,
```

Comment:
Why do we want this to be a property of the `NotePayload` rather than the `Output`?